src/Datadog.Trace.ClrProfiler.Managed.Core/AssemblyInfo.cs

```diff
@@ -1,6 +1,6 @@
 using System.Runtime.CompilerServices;
 
 [assembly: System.Reflection.AssemblyVersionAttribute("1.0.0.0")]
-[assembly: InternalsVisibleTo("Datadog.Trace.ClrProfiler.Managed, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
-[assembly: InternalsVisibleTo("Datadog.Trace.ClrProfiler.IntegrationTests, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
-[assembly: InternalsVisibleTo("Datadog.Trace.ClrProfiler.Managed.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
+[assembly: InternalsVisibleTo("Datadog.Trace.ClrProfiler.Managed, PublicKey=0024000004800000140100000602000000240000525341310008000001000100652bb911930d33a4f67cfd6c712abf77978de155dbdb07b01c08df4f5615af4753b4a5fac5d4d2143ecc8ea6aaab9b3a2fef845a2698139356bf1d1acdd4a4985d493e22b79edbdfc7d1b39de7221669b0b410fa2323a0adc264b953482dea6d913b2c6dcacb3525c3c156f2652df76a3341a39663c9c165ec3cab0f5a80ef72502e49bc5946d23be6c8583ba4a3149f4dbc9a453021e299e5e357846c40514f33db357de4c855dc3225ef96beefe6ee91da24010ea50810f919513756e67e81a363811f8581d5dd0ed4e3e33a725107e70c0e32dcd0e9038885d75f3c0a8c8e0000b98b5ca318827ecc7e5124af2a46c93d6d61c2043c242aafbd33eeb11b87")]
```

Comment:
the public key is longer thanks to https://docs.microsoft.com/en-us/dotnet/standard/assembly/enhanced-strong-naming ?

Reply:
yep, that's what I think